tests/es6.go

```diff
@@ -0,0 +1,34 @@
+/*
+ *
+ * xk6-browser - a browser automation extension for k6
+ * Copyright (C) 2021 Load Impact
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+// Package testutils is indended only for use in tests, do not import in production code!
+package tests
```

Comment:
I commented out because of the linter unused warning. Since I don't know how all this stuff works, I thought that function could be a valuable reference for the future. Do you have another suggestion for saving these as a reference (_other than storing them somewhere on my computer_)?

Reply:
Sure, just store a link to it? :smile:

https://github.com/grafana/k6/blob/cd9469593a0c0fb78541b2ab14466884dba3317f/js/modules/k6/http/request_test.go#L60-L70

Or put that in a comment where you think it might be relevant. Or in a personal documentation system. Or in _the cloud_? :stuck_out_tongue: But committing commented out code that's only useful for you right now doesn't make much sense.